controllers/cohortdata.go

```diff
@@ -69,22 +65,15 @@ func (u CohortDataController) RetrieveHistogramForCohortIdAndConceptId(c *gin.Co
 }
 
 func (u CohortDataController) RetrieveStatsForCohortIdAndConceptId(c *gin.Context) {
-	sourceIdStr := c.Param("sourceid")
-	log.Printf("Querying source: %s", sourceIdStr)
-	cohortIdStr := c.Param("cohortid")
-	log.Printf("Querying cohort for cohort definition id: %s", cohortIdStr)
-	conceptIdStr := c.Param("conceptid")
-	if sourceIdStr == "" || cohortIdStr == "" || conceptIdStr == "" {
-		c.JSON(http.StatusBadRequest, gin.H{"message": "bad request"})
+	sourceId, cohortId, conceptIdsAndCohortPairs, err := utils.ParseSourceIdAndCohortIdAndVariablesAsSingleList(c)
```

Comment:
A suggestion here is to rename `conceptIdsAndCohortPairs` to `conceptDefsAndCohortPairs` to match the function name

Reply:
same for line 76, 86 and 97

Reply:
done